packages/odf/components/kms-config/azure-config.tsx

```diff
@@ -0,0 +1,268 @@
+                  )
+                : validateConnectionName(azureState.name.value, t)}
+            </HelperTextItem>
+          </HelperText>
+        </FormHelperText>
+      </FormGroup>
+
+      <FormGroup
+        fieldId="azure-vault-url"
+        label={t('Azure Vault URL')}
+        labelIcon={
+          <FieldLevelHelp>
+            {t(
+              'This URL is used to access and manage secrets, keys, and certificates stored in Azure Key Vault.'
+            )}
+          </FieldLevelHelp>
+        }
+        className={`${className}__form-body`}
+        isRequired
+      >
+        <TextInput
+          value={azureState.azureVaultURL?.value}
+          onChange={(_event, address: string) => setAzureVaultURL(address)}
+          type="url"
+          id="azure-vault-url"
+          name="azure-vault-url"
+          isRequired
+          validated={getValidatedVaultURLProp}
+          data-test="azure-vault-url-text"
+        />
+        <FormHelperText>
+          <HelperText>
+            <HelperTextItem variant={getValidatedVaultURLProp}>
+              {getValidatedVaultURLProp === ValidatedOptions.error &&
+                validateAzureVaultURLMessage}
+            </HelperTextItem>
+          </HelperText>
+        </FormHelperText>
+      </FormGroup>
+      <FormGroup
+        fieldId="azure-client-id"
+        label={t('Client ID')}
+        labelIcon={
+          <FieldLevelHelp>
+            <Trans t={t}>
+              Client ID is the unique identifier of an application created in
+              Azure AD (Active Directory).
+              <br />
+              <br />
+              You can locate the client ID within Azure Active Directory by
+              accessing the<strong> App registrations</strong> section.
+            </Trans>
+          </FieldLevelHelp>
+        }
+        className={`${className}__form-body`}
+        isRequired
+      >
+        <TextInput
+          value={azureState.clientID?.value}
+          onChange={(_event, clientID: string) => setClientID(clientID)}
+          type="text"
+          id="azure-client-id"
+          name="azure-client-id"
+          isRequired
+          validated={getValidatedClientIDProp}
+          data-test="azure-client-id-text"
+        />
+        <FormHelperText>
+          <HelperText>
+            <HelperTextItem variant={getValidatedClientIDProp}>
+              {getValidatedClientIDProp === ValidatedOptions.error &&
+                validateClientIDMessage}
+            </HelperTextItem>
+          </HelperText>
+        </FormHelperText>
+      </FormGroup>
+      <FormGroup
+        fieldId="azure-tenant-id"
+        label={t('Tenant ID')}
+        labelIcon={
+          <FieldLevelHelp>
+            <Trans t={t}>
+              The tenant ID determines the Azure AD instance where the
+              application is hosted - it is the same as the organization ID.
+              Also referred to as the Microsoft Entra tenant ID.
+              <br />
```

Comment:
```suggestion
              <br />
```
Why to break lines? If you need spaces add margin

Reply:
Sure, updated as per suggestion